core/controllers/library.py

```diff
@@ -215,15 +214,24 @@ def get(self):
             (ord(char), None) for char in string.punctuation)
         query_string = query_string.translate(remove_punctuation_map)
 
-        if self.request.get('category'):
-            query_string += ' category=%s' % self.request.get('category')
-        if self.request.get('language_code'):
-            query_string += ' language_code=%s' % self.request.get(
-                'language_code')
+        # If there is a category parameter, it will be in the following form:
+        #     category=("Algebra" OR "Math")
+        category_string = self.request.get('category', '')
```

Comment:
Should we add a regex check to ensure this else raise invalidInput exception?

Reply:
Sounds good, I added it.